public/src/main.tsx

```diff
@@ -47,20 +45,13 @@ const styles = ({ palette, spacing, mixins }: Theme) => createStyles({
   }
 });
 
-const getLinkPathsAndNames = (): string[][] => {
-  const baseList = [
+const getLinkPathsAndNames = (): string[][] =>
+  [
     ['/', 'Home'],
```

Comment:
Do we need home on the menu? The page is empty...

Reply:
good point!